likelihood_field_particle_filter:
  visualize_rate: 10
  z_hit: 0.5
  z_max: 0.05
  z_random: 0.05
  sigma_hit: 0.2
  max_occ_dist: 3.0
  initial_x: -2.0
  initial_y: 0.5
  initial_th: 0.0
  n_particles: 1000
  # state_variances: [0.05, 0.1, 0.05, 0.1, 0.05, 0.01] #x, x', y, y', th, th'
  # state_variances: [0.2, 0.1, 0.2, 0.1, 0.05, 0.01] # position variance high
  # state_variances: [0.05, 0.1, 0.05, 0.1, 0.5, 0.01] # angle variance high
  # state_variances: [0.5, 0.1, 0.5, 0.1, 0.5, 0.01] # position angle variance high
  state_sigmas: [0.4, 0.2, 0.4, 0.2, 0.2, 0.2] # test variance
